test/t/matches/timeslots_test.exs

```diff
@@ -212,6 +212,19 @@ defmodule T.Matches.TimeslotsTest do
 
       assert length(match_event) == 2
     end
+
+    test "decline", %{profiles: [_p1, p2], match: match} do
```

Comment:
not clear what exactly is tested here from the name

Reply:
doesn't really test much + does not belong here (in case of tests above we added MatchEvent check as extra one, but here nothing besides MatchEvent insertion is tested) => if inserted it would probably go to some separate MatchEventTest.exs